cmake/modules/AddSwift.cmake

```diff
@@ -1774,6 +1787,18 @@ function(_add_swift_executable_single name)
     list(APPEND link_flags "-fuse-ld=gold")
   endif()
 
+  INCLUDE(CheckCXXCompilerFlag)
```

Comment:
I actually don't think this block is required. I merely included it because I think it might be. Someone who understands the CMake scripts a little better might be able to quickly deduce its necessity.